app/controllers/offices_controller.rb

```diff
@@ -22,6 +22,7 @@ def new
   end
 
   def edit
+    @becs = BusinessEntity.where(office_id: @office.id)
```

Comment:
Here you could process the entities to give you a hash of `jurisdiction_id -> bec`, so that you can avoid the sql in the view (commented bellow):

```ruby
@becs = Hash[@office.business_entities.map { |be| [be.jurisdiction_id, be.code] ]
```